pandas/util/testing.py

```diff
@@ -27,7 +27,8 @@
     is_bool, is_categorical_dtype, is_datetime64_dtype, is_datetime64tz_dtype,
     is_datetimelike_v_numeric, is_datetimelike_v_object,
     is_extension_array_dtype, is_interval_dtype, is_list_like, is_number,
-    is_period_dtype, is_sequence, is_timedelta64_dtype, needs_i8_conversion)
+    is_period_dtype, is_scalar, is_sequence, is_timedelta64_dtype,
+    needs_i8_conversion)  # noqa
```

Comment:
why did you change this?

Reply:
´´is_scalar`` wasn't in testing, I need it in my tests and think it's helpful to have in this common testing module.

I need to to for scalar because ``assert np.array([2]) == 2`` passes without an AssertionError...

Reply:
Though I do get an flake8 error for unused import. Trying to resolve that...